sale_order_price_recalculation/models/sale_order.py

```diff
@@ -15,3 +19,10 @@ def action_update_names(self):
         lines = self.mapped("order_line")
         lines._compute_name()
         return True
+
+    def _recalculate_prices(self):
+        _logger.info(
```

Comment:
Do we really want to maybe have a big stack in normal logs ? Maybe debug should be sufficent no ?

Reply:
I think it'll not a big stack because it's only one log triggered by server action.
On my side I never use debug log level because it's too much noise. If you think this log is too noisy, it might  better to remove it completely.
The use case of recalculation, is an exceptional operation, don't you think ?

Here is my opinion, but I'm Ok to apply your final choice cc @CRogos

Reply:
I am fine with removing it. If someone is interested in tracking the changes caused by this action, it could install tracking_manager and track the changes on the fields.

Reply:
Thanks for your quick reply, fixed